core/controllers/admin_test.py

```diff
@@ -978,52 +978,6 @@ def test_update_flag_rules_with_rules_of_non_list_of_dict_type_returns_400(
 
         self.logout()
 
-    def test_update_flag_rules_with_unexpected_exception_returns_500(self):
```

Comment:
Testing this behavior seems valuable. Can we keep this behavioral test coverage?

Reply:
Sure, reverted the deletion.